dubbo-common/src/main/java/org/apache/dubbo/common/Version.java

```diff
@@ -61,6 +66,15 @@ public final class Version {
     static {
         // check if there's duplicated jar
         Version.checkDuplicate(Version.class);
+
+        // get dubbo version and last commit id
+        Properties properties =
+            ConfigUtils.loadProperties(Collections.emptySet(), "META-INF/version");
+
+        VERSION = Optional.ofNullable(properties.getProperty("git.build.version"))
+            .filter(StringUtils::isNotBlank)
+            .orElseGet(() -> getVersion(Version.class, ""));
+        LATEST_COMMIT_ID = Optional.ofNullable(properties.getProperty("git.commit.id.full")).orElse("");
```

Comment:
Catch `Throwable` here to prevent `NoDefError`

Reply:
OK

Reply:
> Catch `Throwable` here to prevent `NoDefError`

I see Catch Throwable in ConfigUtils.loadProperties